doc/nvim-tree-lua.txt

```diff
@@ -568,8 +568,10 @@ Reloads the explorer every time a buffer is written to.
 
 *nvim-tree.sort_by*
 Changes how files within the same directory are sorted.
-Can be one of `name`, `case_sensitive`, `modification_time`, `extension`,
-`filetype` or a function.
+Can be one of `"name"`, `"case_sensitive"`, `"modification_time"`, `"extension"`,
+`"suffix"`, `"filetype"` or a function.
+`"extension"` uses all suffixes e.g. `foo.tar.gz` -> `.tar.gz`
+`"suffix"` uses the last e.g. `.gz`
   Type: `string` | `function(nodes)`, Default: `"name"`
 
   Function may perform a sort or return a string with one of the above
```

Comment:
I removed `suffix` from the node; it's more complex to calculate and users can do it themselves if they feel the need.